src/wagtail_ai/static_src/field_panel/main.css

```diff
@@ -12,13 +12,132 @@
   height: auto;
 }
 
+.tippy-box[data-theme='assistant'] {
+  /* Use fixed-width to avoid layout shifting between states. */
+  width: 350px;
+  background-color: var(--w-color-surface-page);
+  color: inherit;
+  box-shadow: 5px 5px 30px var(--w-color-box-shadow-md);
+  border: 1px solid var(--w-color-border-field-default);
+
+  .tippy-content {
+    padding: 0;
+  }
+}
+
+.wai-dropdown .w-dropdown__content {
+  font-size: 0.875rem;
+  border-radius: 0.1875rem;
+}
+
 .wai-dropdown__item {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
   gap: 0.25rem;
+  color: inherit;
+
+  &:not(:first-child) {
+    border-top: 1px solid var(--w-color-border-furniture);
+  }
+
+  &:hover {
+    background-color: var(--w-color-surface-button-outline-hover);
+    color: var(--w-color-text-link-default);
+  }
 }
 
 .wai-dropdown__description {
   font-weight: normal;
+  color: var(--w-color-text-meta);
+}
+
+.wai-dropdown__intermediary {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  padding: 1rem;
+}
+
+.wai-dropdown__suggestion {
+  padding: 1.25rem;
+  padding-top: 1rem;
+  background-color: var(--w-color-info-50);
+
+  .w-theme-dark & {
+    background-color: color-mix(in srgb, var(--w-color-info-100), #000 50%);
```

Comment:
Based on Ben's new designs, but this should ideally be in core, maybe as a proper value rather than a `color-mix`